src/states/elder/mod.rs

```diff
@@ -1737,6 +1689,40 @@ impl Approved for Elder {
 
         Ok(())
     }
+
+    // Checking members vote status and vote to remove those non-resposive nodes.
+    #[cfg(feature = "mock_parsec")]
+    fn check_voting_status(&mut self) {
+        let unresponsive_nodes = self.chain.check_vote_status();
+        for pub_id in unresponsive_nodes.iter() {
+            self.vote_for_event(AccumulatingEvent::Offline(*pub_id));
+        }
+    }
+
+    #[cfg(not(feature = "mock_parsec"))]
+    fn check_voting_status(&mut self) {
+        let unresponsive_nodes = self.chain.check_vote_status();
+        for (id, count) in mem::replace(&mut self.unresponsive_nodes, Default::default()) {
+            if unresponsive_nodes.contains(&id) {
+                let new_count = count.saturating_add(1);
+                info!(
+                    "{} detected {:?} as unresponsive for {} times when having {} members",
+                    self,
+                    id,
+                    new_count,
+                    self.chain.our_info().len(),
+                );
+                if new_count > self.chain.our_info().len() {
```

Comment:
I'm not clear why we need the node to be accused more times than there is elders in the section. Maybe add a comment explaining that?

Reply:
The average chance that the missing peer will pick usas gossip recipient, so that we can know its votes (or we know its votes via others) is 1/number_of_peers.
So requiring `the node to be accused more times than there is elders in the section` is mainly just to ensure the full knowledge is reached before making the accuse.

Reply:
OK, could you add it as comment there?

Reply:
ok, will do